src/app.js

```diff
@@ -1,21 +1,32 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const urlPath = req.url;
+
+  if (urlPath.startsWith('/file/') && urlPath.length > 6) {
+    const filePath = path.join(__dirname, 'public', urlPath.substring(6));
+
+    fs.readFile(filePath, (err, data) => {
+      if (err) {
+        res.writeHead(404, { 'Content-Type': 'text/plain' });
+        res.end('File not found');
+      } else {
+        res.writeHead(200, { 'Content-Type': 'text/html' });
+        res.end(data);
+      }
+    });
+  } else {
+    res.writeHead(200, { 'Content-Type': 'text/html' });
```

Comment:
```suggestion
    res.writeHead(400, { 'Content-Type': 'text/html' });
```
It's a bad user request code, we can set it if we don't accept requests to other endpoints